site-src/blog/2023/0829-mesh-support.md

```diff
@@ -0,0 +1,210 @@
+restarting on Gateway API _conformance profiles_, as discussed in [GEP-1709].
+
+The basic idea of conformance profiles is that we can define subsets of the
+Gateway API, and allow implementations to choose (and document) which subsets
+they conform to. GAMMA is adding a new profile, named `Mesh` and described in
+[GEP-1686], which checks only the mesh functionality as defined by GAMMA. At
+this point, Kuma 2.3+, Linkerd 2.14+, and Istio 1.16+ are all conformant with
+the `Mesh` profile.
+
+## What else is in Gateway API v0.8.0?
+
+This release is all about preparing Gateway API for the upcoming v1.0 release
+where HTTPRoute, Gateway, and GatewayClass will graduate to GA. There are two
+main changes related to this: CEL validation and GEP process changes.
+
+### CEL Validation
+
+The first major change is that Gateway API v0.8.0 is the start of a transition
+from webhook validation to [CEL validation][cel] using information built into
+the CRDs. That will mean different things depending on the version of
+Kubernetes you're using:
+
+#### Kubernetes 1.25+
+
+CEL validation is fully supported, and almost all validation is implemented in
+CEL. (The sole exception is that header names in header modifier filters can
+only do case-insensitive validation. There is more information in [issue
+2277].)
+
+We recommend _not_ using the validating webhook on these Kubernetes versions.
+
+#### Kubernetes 1.23 and 1.24
+
+CEL validation is not supported, but Gateway API v0.8.0 CRDs can still be
+installed. When you upgrade to Kubernetes 1.25+, the validation included in
+these CRDs will automatically take effect.
+
+We recommend continuing to use the validating webhook on these Kubernetes
+versions.
+
+#### Kubernetes 1.22 and older
+
+Gateway API only commits to support for [5 most recent versions of
+Kubernetes][supported-versions]. As such, these versions are no longer
+supported by Gateway API, and unfortunately Gateway API v0.8.0 cannot be
+installed on them, since CRDs containing CEL validation will be rejected.
+
+### GEP Process Changes
+
+The second significant change in Gateway API v0.8.0 is that we have (by
+necessity) taken a hard look at the process around [Experimental][status]
+GEPs. Some of these GEPs have been lingering long enough that projects have
+come to rely on them in production use, which is a bit of a breakdown of the
+GEP process. In order to prevent it happening in the future, we have changed
+the GEP process such that reaching [Experimental][status] _requires_ that a
+GEP include both the graduation criteria by which the GEP will become
+[Standard][status], and a probationary period after which the GEP will be
+dropped if does not meet its graduation criteria.
+
+For an exhaustive list of changes included in the `v0.8.0` release, please see
+the [v0.8.0 release notes]. For more information on Gateway API versioning,
+refer to the [official documentation][versioning docs].
```

Comment:
I don't feel like this is part of the release, it's a change we made to our process, but I don't think we need to highlight it here. I think most readers, at least on kubernetes.io blog will not be familiar enough with Gateway API processes for this to be a meaningful change for them.

Reply:
I don't actually feel that strongly here though. I do get the value of highlighting the potential for things to change since this is about GAMMA being experimental, I just don't think of this as being related to our release process/artifacts.

Reply:
I ended up leaving it: I really hope that the people making GEPs are reading the Gateway API blog, and I think it's important to have this in front of them.